### ISSUE TITLE
early stop？

### ISSUE THREAD

**Issue by balabalaboy** [ZENGYIMING-EAMON/CorrNet3D#13]
How many epochs did you train?
It seems that it takes a long time to train.
It seems that the decreasing trend of loss has levelled off, and the loss value increases instead of decreasing. Then roughly how many epochs will the training stop?
![image](https://user-images.githubusercontent.com/125865806/226239391-bc7d1c44-d3f8-4e62-bd5d-4b7f166937f3.png)
![image](https://user-images.githubusercontent.com/125865806/226239494-a931f1f4-fe25-4bc0-b7b6-414161f7a700.png)

![image](https://user-images.githubusercontent.com/125865806/226239423-c2522bf8-0ef8-418c-a6aa-083dbd5cddff.png)

By the way, is it possible to register one could point to another cloud point single time? not the total dataset, just the single one.

**Comment by TAO-TAO-TAO-TAO-TAO** [comment]
> How many epochs did you train? It seems that it takes a long time to train. It seems that the decreasing trend of loss has levelled off, and the loss value increases instead of decreasing. Then roughly how many epochs will the training stop? ![image](https://user-images.githubusercontent.com/125865806/226239391-bc7d1c44-d3f8-4e62-bd5d-4b7f166937f3.png) ![image](https://user-images.githubusercontent.com/125865806/226239494-a931f1f4-fe25-4bc0-b7b6-414161f7a700.png)
> 
> ![image](https://user-images.githubusercontent.com/125865806/226239423-c2522bf8-0ef8-418c-a6aa-083dbd5cddff.png)
> 
> By the way, is it possible to register one could point to another cloud point single time? not the total dataset, just the single one.

I would like to ask how long it will take you to train for approximately 100 epochs. I need over half an hour to train an epoch, using 12g of 2080ti

**Comment by black-prince222** [comment]
> How many epochs did you train? It seems that it takes a long time to train. It seems that the decreasing trend of loss has levelled off, and the loss value increases instead of decreasing. Then roughly how many epochs will the training stop? ![image](https://user-images.githubusercontent.com/125865806/226239391-bc7d1c44-d3f8-4e62-bd5d-4b7f166937f3.png) ![image](https://user-images.githubusercontent.com/125865806/226239494-a931f1f4-fe25-4bc0-b7b6-414161f7a700.png)
> 
> ![image](https://user-images.githubusercontent.com/125865806/226239423-c2522bf8-0ef8-418c-a6aa-083dbd5cddff.png)
> 
> By the way, is it possible to register one could point to another cloud point single time? not the total dataset, just the single one.

I have the same question. my rank loss and val loss is increasing.

**Comment by black-prince222** [comment]
> How many epochs did you train? It seems that it takes a long time to train. It seems that the decreasing trend of loss has levelled off, and the loss value increases instead of decreasing. Then roughly how many epochs will the training stop? ![image](https://user-images.githubusercontent.com/125865806/226239391-bc7d1c44-d3f8-4e62-bd5d-4b7f166937f3.png) ![image](https://user-images.githubusercontent.com/125865806/226239494-a931f1f4-fe25-4bc0-b7b6-414161f7a700.png)
> 
> ![image](https://user-images.githubusercontent.com/125865806/226239423-c2522bf8-0ef8-418c-a6aa-083dbd5cddff.png)
> 
> By the way, is it possible to register one could point to another cloud point single time? not the total dataset, just the single one.


Hello, do you know why the loss is an uptrend? I also encountered this situation.

**Comment by eashratj** [comment]
@black-prince222 @balabalaboy  do you have any answer by now ? : )  Until how many epoch it stopped? It would be really helpful if you answer!
